### PR TITLE
97 fix error on registration

### PR DESCRIPTION
**Pull request by levmyshkin**
Added patch for drupal core:
https://www.drupal.org/project/drupal/issues/3226117

Added admin email "admin@drupal.am"

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -46,15 +46,15 @@
...
+        "drupal/core": "^9.3.14",
+        "drupal/core-composer-scaffold": "^9.3.14",
+        "drupal/core-project-message": "^9.3.14",
+        "drupal/core-recommended": "^9.3.14",
         "drupal/ctools": "^3.4",
         "drupal/default_content": "^2.0@alpha",
         "drupal/default_content_deploy": "^1.0@beta",
         "drupal/ds": "^3.5",
-        "drupal/email_registration": "^1.1",
+        "drupal/email_registration": "^1.x-dev",
```
Comment (leo72) [inline composer.json:57]: Not sure we should use dev version. Maybe it's OK, but need to see the diff.

File: config/sync/user.mail.yml [hunk 2]
```diff
@@ -3,115 +3,28 @@ _core:
 langcode: hy
 cancel_confirm:
   subject: 'Account cancellation request for [user:display-name] at [site:name]'
-  body: |-
```
Comment (leo72) [inline config/sync/user.mail.yml:6]: Is it desired intended change? Or just the result of saving the form?

**Review by leo72 — CHANGES_REQUESTED** [review]

File: config/sync/user.mail.yml (hunk 2 above)
Comment (levmyshkin) [reply config/sync/user.mail.yml:6]: Yes, it's the result of saving Account settings form. There are not actual changes only another approach for line endings.

Comment (leo72) [reply config/sync/user.mail.yml:6]: Let's exclude this from PR now.

File: composer.json (hunk 1 above)
Comment (levmyshkin) [reply composer.json]: I pushed it with other branch and it's already in master:
354927acfe0a045ad629405399f6b246f3dc3e06
86 - Use stable version of Email Registration

I merged master in this pull request

**Pull request merged**